Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -65,17 +65,25 @@ Kindly consult [GSFDICT] for more definitions used in this document.
 
 ## Introduction
 
-#### Problem Statement
+What you cannot measure, you cannot improve.
```

Comment:
```suggestion
> "If you can't measure it, you can't improve it." - Peter Drucker
```

Reply:
Correcting the quote and adding in the attribution, especially since this is the opening line of the SCI